### ISSUE TITLE
Not showing streak again

### ISSUE THREAD

**Issue by spences10** [Naramsim/GithubOriginalStreak#92]
Referencing: https://github.com/Naramsim/GithubOriginalStreak/issues/89

Looks like this has stopped doing it's thing again.

Is this a known issue?

I've tried on Edge and Firefox and still showing 366 days as longest streak

**Comment by Naramsim** [comment]
It seems only happening for you @spences10 

Tomorrow I'll release a version that doesn't update the database file, so we won't have anymore these issues.

**Comment by spences10** [comment]
Oh! Is there anything I can do to debug it? I've tried uninstalling and reinstalling, and it's for both Firefox and Edge?

Is it related to my profile then do you think?

**Comment by Naramsim** [comment]
Hi @spences10 

Yesterday I released a version that fixes it on Chrome. 0.15.0 Today I'll release it on FF and Opera.

Anyway, the issue was caused by the Github API which truncates the list of files present in a Gist to 300 files. https://docs.github.com/en/rest/reference/gists#truncation

I thus changed the logic of the application. I transferred all the files here on this repo and made them read-only. https://github.com/Naramsim/GithubOriginalStreak/blob/master/database/spences10
